addons/event/data/email_template_data.xml

```diff
@@ -281,5 +281,159 @@
             ]]></field>
         </record>
 
+        <record id="event_change_notification" model="mail.template">
+            <field name="name">Event: schedule change</field>
+            <field name="model_id" ref="event.model_event_registration"/>
+            <field name="email_from">${(object.event_id.organizer_id.email or object.event_id.user_id.email or '')|safe}</field>
+            <field name="email_to" >${object.partner_id.email or object.email|safe}</field>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="reply_to">${object.event_id.reply_to|safe}</field>
+            <field name="subject">
+%if object.event_id.state == 'confirm'
+Schedule of event ${object.event_id.name} changed.
+%endif
+%if object.event_id.state == 'cancel'
+Event ${object.event_id.name} is cancelled.
+%endif
+</field>
+            <field name="body_html"><![CDATA[
+% set date_begin = format_tz(object.event_id.date_begin, tz='UTC', format='%Y%m%dT%H%M%SZ')
+% set date_end = format_tz(object.event_id.date_end, tz='UTC', format='%Y%m%dT%H%M%SZ')
+<div style="background:#F3F5F6;color:#515166;padding:25px 0px;font-family:Arial,Helvetica,sans-serif;font-size:14px;">
+    <table style="width:600px;margin:5px auto;">
+        <tbody>
+            <tr>
+                <td>
+                    <a href="/">
```

Comment:
because of this  : 

https://github.com/odoo/odoo/blob/master/addons/mail/models/mail_template.py#L302

it will find href then replate it with base url.